Dockerfile

```diff
@@ -8,25 +11,21 @@ RUN useradd osg \
        yumrepo=osg-upcoming; else \
        yumrepo=osg-upcoming-$BASE_YUM_REPO; fi \
  && mkdir -p ~osg/.condor \
- && yum -y --enablerepo=$yumrepo install condor \
+ && yum -y --enablerepo=$yumrepo install \
+        condor \
+        osg-wn-client \
+        redhat-lsb-core \
+        singularity \
  && yum clean all \
  && mkdir -p /etc/condor/passwords.d /etc/condor/tokens.d \
  && curl -s -o /usr/sbin/osgvo-user-job-wrapper https://raw.githubusercontent.com/opensciencegrid/osg-flock/master/job-wrappers/user-job-wrapper.sh \
  && curl -s -o /usr/sbin/osgvo-node-advertise https://raw.githubusercontent.com/opensciencegrid/osg-flock/master/node-check/osgvo-node-advertise \
  && chmod 755 /usr/sbin/osgvo-user-job-wrapper /usr/sbin/osgvo-node-advertise
 
-COPY entrypoint.sh /bin/
+COPY 10-setup-htcondor.sh /etc/osg/image-init.d/
+COPY 10-cleanup-htcondor.sh /etc/osg/image-cleanup.d/
```

Comment:
I was not familiar with the semantics of the image init / cleanup .d dirs, but that seems like a nice separation.